operator/cnp_event.go

```diff
@@ -89,6 +89,8 @@ func enableCNPWatcher() error {
 			AddFunc: func(obj interface{}) {
 				metrics.EventTSK8s.SetToCurrentTime()
 				if cnp := k8s.ObjToSlimCNP(obj); cnp != nil {
+
+
```

Comment:
Was this on purpose? I'm fine with it but it might not pass gofmt? (mine usually collapses multiple newlines)

Reply:
I missed this, it is fixed now.